tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -117,11 +117,11 @@ exports[`bring_local_self.w 1`] = `
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Cannot bring a module into itself
 
 
-error: Cannot find module \\"../../../examples/tests/invalid/non-existent.w\\"
+error: Cannot find module \\"./non-existent.w\\"
```

Comment:
Regression I think, the path shown is relative to cwd

Reply:
Nice, I like that idea.
Showing the absolute path in the hint might be good too. Usual absolute paths aren't good in errors but seems relevant for this.

Reply:
https://github.com/winglang/wing/issues/4503